dts/arm/nxp/nxp_mcxn94x_common.dtsi

```diff
@@ -850,6 +850,15 @@
 		num-bidir-endpoints = <8>;
 		status = "disabled";
 	};
+
+	lptmr0: lptmr@4a000 {
+		compatible = "nxp,kinetis-lptmr";
```

Comment:
Can it be "nxp,lptmr" without kinetis?

Reply:
Unfortunately no, not unless we change the compatible string across all parts. Typically we have left compatibles with the family name as is to avoid churn for downstream driver users, although we want to avoid adding new drivers with the family name (so this issue does not occur in the future)

Reply:
hope, we will be brave and clean it up sooner.